pkg/network/protocols/kafka/telemetry.go

```diff
@@ -28,8 +28,8 @@ func NewTelemetry() *Telemetry {
 
 	return &Telemetry{
 		metricGroup:    metricGroup,
-		produceHits:    newAPIVersionCounter(metricGroup, "total_hits", MinSupportedProduceRequestApiVersion, MaxSupportedProduceRequestApiVersion, "operation:produce", libtelemetry.OptStatsd),
-		fetchHits:      newAPIVersionCounter(metricGroup, "total_hits", MinSupportedFetchRequestApiVersion, MaxSupportedFetchRequestApiVersion, "operation:fetch", libtelemetry.OptStatsd),
+		produceHits:    newAPIVersionCounter(metricGroup, "total_hits", ClassificationMinSupportedProduceRequestApiVersion, DecodingMaxSupportedProduceRequestApiVersion, "operation:produce", libtelemetry.OptStatsd),
+		fetchHits:      newAPIVersionCounter(metricGroup, "total_hits", ClassificationMinSupportedFetchRequestApiVersion, DecodingMaxSupportedFetchRequestApiVersion, "operation:fetch", libtelemetry.OptStatsd),
```

Comment:
Same as above

Reply:
Weird, must have fixed it in another branch
Fixed